plugins/calibrecompanion.koplugin/main.lua

```diff
@@ -23,6 +23,8 @@ require("ffi/zeromq_h")
 --]]
 local CalibreCompanion = InputContainer:new{
     name = "calibrecompanion",
+    fullname = _("Calibre companion"),
+    description = _([[Send documents from Calibre library directly to device via Wifi connection]]),
```

Comment:
calibre (lowercase c)
Wi-Fi (I vote wifi but at some point we decided on Wi-Fi)